src/libxively/tls/certs/xi_RootCA_list.h

```diff
@@ -12,10 +13,10 @@ extern "C" {
 #endif
 
 #ifndef XI_ROOTCA_LIST_BYTE_LENGTH
-#define XI_ROOTCA_LIST_BYTE_LENGTH 5941
-#endif
+#define XI_ROOTCA_LIST_BYTE_LENGTH 5942
+#endif /* XI_ROOTCA_LIST_BYTE_LENGTH */
 
-extern const unsigned char xi_RootCA_list[XI_ROOTCA_LIST_BYTE_LENGTH];
+extern unsigned char xi_RootCA_list[ XI_ROOTCA_LIST_BYTE_LENGTH ];
```

Comment:
why not const?

Reply:
because mbedTLS BSP has to alter last character?

Reply:
That's right. The application would crash with a `SIGBUS` error otherwise.